### DISCUSSION TITLE
Delete branch

### DISCUSSION THREAD

**Discussion by BhJaipal** [community/community#46841] [Mobile]
I want to delete a branch, how do I do it with github.com

**Comment by jge162** [comment]
Hello @BhJaipal Go to your repository on GitHub.
- Click on the "Branches" tab.
- Locate the branch you want to delete and click on it.
- Click the "Delete" button.
- Confirm the deletion by typing the name of the branch you want to delete and clicking "Delete branch."
This should work. If not please let me know. Thanks.

**Reply by BhJaipal** [reply]
I can't find delete button 
Repo main page or in settings of repo

**Reply by jge162** [reply]
If you click on branches tab, then view branches. A list should show up. On the right dude you will see a trash can icon. Try that

**Reply by jge162** [reply]
You can’t do this on mobile. Please be on a computer.

**Reply by BhJaipal** [reply]
I don't have computer

**Reply by jge162** [reply]
That’s why you can’t see it. You need a desktop view.

**Reply by SilasRodrigues19** [reply]
Hey @BhJaipal 

Enable the option `Desktop View / Desktop Site`  then, access the repo and put `/banches` at the end of the URL.

Example: https://github.com/BhJaipal/Javascript_use/branches

A trash icon will appear to be able to delete the branch

**Reply by BhJaipal** [reply]
Thank you @jge162 and @SilasRodrigues19

**Reply by jge162** [reply]
I just checked. If you turn the phone landscape you can see the delete button. See pic below. 
![440741EF-BF0B-485A-8A58-58545927CA33](https://user-images.githubusercontent.com/31228460/217879618-6e75864e-dc05-4868-b51c-278d42786a96.png)
